### DISCUSSION TITLE
Dismiss lazygit's own popup without closing entire lazygit session?

### DISCUSSION THREAD

**Discussion by adamdicarlo0** [LazyVim/LazyVim#370] [Q&A] [answered]
Hi!

I'm new to both LazyVim and lazygit. (Both _wonderful_ pieces of software! :heart:)

When I use lazygit via `<leader>gg`, I can't seem to send `<ESC>` to lazygit (to e.g. abort its commit message popup). My keystroke goes to NeoVim instead, leaving insert mode; then, if I press `<ESC>` again, the lazygit session is killed.

How is one supposed to send `<ESC>` to lazygit?

**Comment by folke** [comment, marked as answer]
Double escape is indeed to enter normal mode in the terminal.

Pressing escape once works as expected for me.

Did you maybe change timeoutlen? And did you set that to a high number?

If that's the case, then the single escape will only trigger once timeoutlen expires

**Comment by adamdicarlo0** [comment]
Wow, accidentally deleted my comment 😭 

`timeoutlen` was not the problem, but your response led me to the answer! I was assuming the (small amount) of my own custom config beyond the LazyVim starter was surely not the problem 😅 😆

I had this:

```
vim.keymaps.set("t", "<esc>", "<c-\\><c-n>")
```

which I'd adapted from my old long-in-the-tooth vimscript NeoVim config.

So thank you for the help!

And as I tried to post the first time, thank you for LazyVim :heart: -- it's amazing!

**Reply by adamdicarlo0** [reply]
First time posting on a GitHub discussion. Looks like it has a bug, showing your comment twice after posting... I thought I'd doubled posted so I tried to delete one. 😢  I see now, also, that there's a reply box I should have used to reply to folke.

**Reply by folke** [reply]
Good to know you have it fixed!
